### ISSUE TITLE
No Games Found text position

### ISSUE THREAD

**Issue by D3SOX** [flavioislima/HeroicGamesLauncher#175]
**Describe the bug**
The `No Games Found` text is not in the same position as the `Total Games` text

**To Reproduce**
1. Filter for `Not Ready`
2. Filter for `Downloading` (while not downloading anything)
3. Compare the positions

**Expected behavior**
It should be in the same position as the `Total Games` text

**Screen records**

https://user-images.githubusercontent.com/24937357/108406514-d3a20f80-7222-11eb-82ad-54de823571cb.mp4

https://user-images.githubusercontent.com/24937357/108406520-d6046980-7222-11eb-9eb2-af004c905898.mp4

**Desktop (please complete the following information):**
 - OS: Arch Linux
 - Version: 1.5.2 (from AUR `heroic-games-launcher-bin`)

**Comment by flavioislima** [comment]
Because it is not the same element. 
Btw, this is something you could fix with a PR since you're a dev as well 😛

**Comment by D3SOX** [comment]
I'm not that familiar with React, more with Vue, but I can take a look :smiley:

**Comment by D3SOX** [comment]
I'm still having issues setting up the development environment. Could you help me out here please:
![a](https://user-images.githubusercontent.com/24937357/108481198-2d90ec80-7298-11eb-870f-5e3099a38dde.png)

**Comment by flavioislima** [comment]
Nice. I'll create a discussion channel here on GitHub that will make it easier for us to discuss and help each other with the development.

**Comment by flavioislima** [comment]
> I'm still having issues setting up the development environment. Could you help me out here please:
> ![a](https://user-images.githubusercontent.com/24937357/108481198-2d90ec80-7298-11eb-870f-5e3099a38dde.png)

Oh, I think I have to update the ReadMe then. instead of running yarn start. Open two consoles and run:
`yarn react-start `on one and `yarn electron` on the other. I'm doing this way because it is easier to debug the electron part this way.
But I'll fix the start script soon as well.

**Comment by D3SOX** [comment]
Off topic: I think you should stick to either `yarn`  or `npm` (I personally prefer `yarn`)

**Comment by wiryfuture** [comment]
Yarn 2 sounds interesting to use

**Comment by flavioislima** [comment]
I want to use Yarn now. I used it for a long time but then Yarn 2 came and I hated it. But it is been some time since I tried it.
Npm 7 is also very good and compatible with Yarn files. 

For now, let's use Yarn 1 and I'll make tests with Yarn 2 to check if it is worth it. 😄 

So let's just avoid committing package-lock and yarn-lock if there are no new or updated packages to avoid merge conflicts. 👍🏽
